src/index.ts

```diff
@@ -30,24 +30,24 @@ const rimrafVersion = '^6.0.1';
 const rollupVersion = '^4.30.1';
 let updatePrettierJava = false;
 const variables = {
-  minSdkVersion: 23,
-  compileSdkVersion: 35,
-  targetSdkVersion: 35,
-  androidxActivityVersion: '1.9.2',
-  androidxAppCompatVersion: '1.7.0',
-  androidxCoordinatorLayoutVersion: '1.2.0',
-  androidxCoreVersion: '1.15.0',
-  androidxFragmentVersion: '1.8.4',
+  minSdkVersion: 24,
+  compileSdkVersion: 36,
+  targetSdkVersion: 36,
+  androidxActivityVersion: '1.11.0',
+  androidxAppCompatVersion: '1.7.1',
+  androidxCoordinatorLayoutVersion: '1.3.0',
+  androidxCoreVersion: '1.17.0',
+  androidxFragmentVersion: '1.8.9',
   firebaseMessagingVersion: '24.1.0',
   playServicesLocationVersion: '21.3.0',
   androidxBrowserVersion: '1.8.0',
   androidxMaterialVersion: '1.12.0',
   androidxExifInterfaceVersion: '1.3.7',
   coreSplashScreenVersion: '1.0.1',
```

Comment:
Believe we can use the latest dependencies here:

```suggestion
  firebaseMessagingVersion: '25.0.1',
  playServicesLocationVersion: '21.3.0',
  androidxBrowserVersion: '1.9.0',
  androidxMaterialVersion: '1.13.0',
  androidxExifInterfaceVersion: '1.4.1',
  coreSplashScreenVersion: '1.2.0',
```